user-info-provider/README.md

```diff
@@ -135,3 +135,57 @@ This mapper produces the following standardized metadata JSON-structure
 }
 ```
 
+## Configuration based on role
+There is now a possibility to hide the person tag in the claim based on the role attribute which the user has. If this
+needs to be used please add the professional roles which need to be checked in the mapper and select the negate output
+accordingly.
+
+<img src="../docs/userinfo/Vidis-role-based-info-mapper.png" width="70%"/>
+
+# Role based info Mapper
+
+Service Provider Mapper similar to the user property mapper provided by Keycloak, which reads provided property from the
+user attribute based on the role which is assigned to the user and adds them to the token. Role is checked in the following
+user attributes : rolle, person.kontext.rolle, person.kontext.<number>.rolle.
```

Comment:
isn't it better to have them configurable? or is it part of the "standard"?

Reply:
its the "Sanis" API where this is from right? I would add that to the name of the mapper and also add this info about "Role is checked in the following user attributes" in the mappers config tooltip!

Reply:
Making it configurable would also make it too complex. So it was decided to keep it in code for now rather than keeping it configurable. Please see the comments in the ticket.
https://intension-de.atlassian.net/browse/FWUV-236
Also, it is not always fetched from SANIS API. Michael knows more details about it.